spec/abci++/abci++_tmint_expected_behavior.md

```diff
@@ -216,3 +216,7 @@ As for the new methods:
   Legacy applications looking to reuse old code that implemented `DeliverTx` should wrap the legacy
```

Comment:
I could not comment the previous line, but `Commit` should be removed from there!

Reply:
Oops!